O/OpenFHE/build_tarballs.jl

```diff
@@ -1,72 +1,13 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using BinaryBuilder, Pkg
+
+# If you make changes to this file, be sure to update ../OpenFHE_128/build_tarballs.jl as well.
```

Comment:
I would put the comment next to the `name = ...` line, since the `common.jl` file should not be changed, right? Also, maybe make the motivation more explicit:
```julia
# If you make changes in this file, e.g., to release a new version,
# be sure to also release a new version of `OpenFHE_128` as well (see `../OpenFHE_128/build_tarballs.jl`)
```